Configuration/Eras/python/Era_Phase2C10_dd4hep_noMkFit_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Phase2C10_dd4hep_cff import Phase2C10_dd4hep
+from Configuration.Eras.ModifierChain_trackingMkFitProd_cff import trackingMkFitProdPhase2
+
+Phase2C10_dd4hep_noMkFit = cms.ModifierChain(Phase2C10_dd4hep.copyAndExclude([trackingMkFitProdPhase2]))
```

Comment:
I would've made a `noMkFit` only for the production detector setup. But I guess it wouldn't really hurt to have more.

Reply:
I am aware I probably overdid it, but I thought that overdoing was better than the other way around, to avoid potential objections, as indeed it shouldn't hurt to have more.